lmfdb/backend/statstable.py

```diff
@@ -753,6 +753,62 @@ def min(self, col, constraint={}, record=True):
                 self._record_extreme(col, ccols, cvals, m, kind="min")
         return m
 
+    def _slow_sum(self, col):
+        """
+        Compute the sum of a column
+
+        INPUT:
+
+        - ``col`` -- the column
+        """
+        inserter = SQL("SELECT SUM({0}) FROM {1}")
+        inserter = inserter.format(Identifier(col), Identifier(self.search_table))
+        cur = self._execute(inserter)
+        if cur.rowcount:
+            return cur.fetchone()[0]
+
+    def _record_sum(self, col, m):
+        """
+        Cache the sum
+
+        INPUT:
+
+        - ``col`` -- the column
+        - ``m``   -- the value to store as sum
+        """
+        inserter = SQL(
+                "INSERT INTO {0} "
+                "(cols, value) "
+                "VALUES (%s, %s)"
```

Comment:
You need to include the kind of statistic as well as constraint columns.  See `_record_extreme` for an example.

Reply:
done; actually i just used the existing `_record_extreme` which i renamed to `_record_statistic` because it's not just for min and max anymore